src/libraries/System.Runtime.Caching/ref/System.Runtime.Caching.csproj

```diff
@@ -1,10 +1,16 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);netcoreapp3.1;netstandard2.0</TargetFrameworks>
```

Comment:
To make sure that P2Ps pick up this configuration in the ref when P2Pin the source, please also add a rid-less `$(NetCoreAppCurrent)` configuration to the src. Otherwise when a project P2Ps the source from another NetCoreAppCurrent (rid-less) project, the netcoreapp3.1 src would be picked which references the netcoreapp3.1 ref. To stay consistent, we want to add the `NetCoreAppCurrent` configuration to the source so that the `NetCoreAppCurrent` ref is referenced.